packages/experiments/src/components/Card/Card.types.ts

```diff
@@ -0,0 +1,110 @@
+import { ICardHeaderProps } from './CardHeader/CardHeader.types';
+import { ICardContentDetails } from './Layout/Layout.types';
+import { IAction } from './ActionBar/ActionBar.types';
+import { ICardDropDownOption } from './CardFrame/CardFrame.types';
+
+/**
+ * Card size that we want to build.
+ */
+export enum CardSize {
+  /**
+   * Option for selecting small card
+   */
+  small,
+
+  /**
+   * Option for selecting Medium Tall card
+   */
+  mediumTall,
+
+  /**
+   * Option for selecting Medium Wide card
+   */
+  mediumWide,
+
+  /**
+   * Option for selecting Large card
+   */
+  large
+}
+
+/**
+ * Card content type that we want to display in the card content area
+ */
+export enum CardContentType {
+  /**
+   * Selects BodyText component
+   */
+  BodyText,
+
+  /**
+   * Selects Thumbnail List component
+   */
+  ThumbnailList,
+
+  /**
+   * Selects the compound button stack
+   */
+  CompoundButtonStack
+}
+
+/**
+ * Priority on the card.
+ */
+export enum Priority {
+  /**
+   * Renders content in the content area 1. For small card always choose this priority
+   */
+  Priority1 = 0,
+
+  /**
+   * Renders content in the content area 2.
+   */
+  Priority2
+}
+
+export interface ICardFrameContent {
+  /**
+   * Cards title
```

Comment:
>Cards [](start = 5, length = 5)

Just Card

Reply:
fixed

---
In reply to: [195604219](https://github.com/OfficeDev/office-ui-fabric-react/pull/5196#discussion_r195604219) [](ancestors = 195604219)